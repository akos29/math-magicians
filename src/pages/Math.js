import React from 'react';
import Calculator from '../components/Calculator';

function Math() {
  return (
    <div className="calc-container">
      <h2>Let&apos;s do some math</h2>
      <Calculator />
    </div>
  );
}

export default Math;
