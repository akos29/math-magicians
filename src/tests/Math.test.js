import React from 'react';
import TestRenderer from 'react-test-renderer';
import Math from '../pages/Math';

it('renders correctly', () => {
  const testRenderer = TestRenderer.create(<Math />).toJSON();
  expect(testRenderer).toMatchSnapshot();
});
