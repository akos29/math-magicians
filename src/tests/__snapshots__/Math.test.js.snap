// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="calc-container"
>
  <h2>
    Let's do some math
  </h2>
  <div
    className="calc"
  >
    <form
      className="row-group"
    >
      <input
        className="result"
        disabled={true}
        value={0}
      />
    </form>
    <div
      className="row-group"
    >
      <button
        className="keys"
        id="AC"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="keys"
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className="keys"
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="keys"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
    </div>
    <div
      className="row-group"
    >
      <button
        className="keys"
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className="keys"
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className="keys"
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="keys"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
    </div>
    <div
      className="row-group"
    >
      <button
        className="keys"
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className="keys"
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className="keys"
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="keys"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
    </div>
    <div
      className="row-group"
    >
      <button
        className="keys"
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className="keys"
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className="keys"
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="keys"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
    </div>
    <div
      className="row-group"
    >
      <button
        className="keys zero"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        className="keys"
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="keys"
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</div>
`;
